render.yaml

```diff
@@ -27,11 +27,38 @@ services:
         - docs/**
         - '**/*.spec.ts'
 
+  - type: web
+    name: toolpad-demo
+    env: docker
+    branch: demo
+    dockerfilePath: ./docker/images/toolpad-demo/Dockerfile
+    dockerContext: ./docker/images/toolpad-demo/
+    healthCheckPath: /health-check
+    plan: starter plus
+    pullRequestPreviewsEnabled: false
+    envVars:
```

Comment:
Shouldn't we set the demo mode env var?

Reply:
Yeah good one, I'll set every env var except the secret ones

Reply:
I guess just `TOOLPAD_DEMO` should do

Reply:
Yep, as much infrastructure expressed in code as possible